### PR TITLE
KAFKA-4658: Improve test coverage InMemoryKeyValueLoggedStore

### PR DESCRIPTION
**Pull request by jeyhunkarimov**

### CODE REVIEW

**Comment by jeyhunkarimov** [comment]
call for review, @mjsax @dguy

File: streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java [hunk 1]
```diff
@@ -46,4 +52,27 @@
...
+        List<KeyValue<Integer, String>> entries = new ArrayList<>();
+        entries.add(new KeyValue<Integer, String>(1, "1"));
+        entries.add(new KeyValue<Integer, String>(2, "2"));
+        store.putAll(entries);
+        assertEquals(store.get(1), "1");
+        assertEquals(store.get(2), "2");
+    }
+
+    @Test
+    public void testIfPersistent() {
```
Comment (mjsax) [inline streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java]: test name: `shouldCreatePersistentStore` ?

**Review by mjsax — COMMENTED** [review]
Just a naming nit :)

**Review by mjsax — APPROVED** [review]
LGTM.

Call for final review and merging @guozhangwang

File: streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java [hunk 2]
```diff
@@ -46,4 +52,27 @@
...
+        List<KeyValue<Integer, String>> entries = new ArrayList<>();
+        entries.add(new KeyValue<Integer, String>(1, "1"));
+        entries.add(new KeyValue<Integer, String>(2, "2"));
+        store.putAll(entries);
+        assertEquals(store.get(1), "1");
+        assertEquals(store.get(2), "2");
+    }
+
+    @Test
+    public void shouldCreatePersistentStore() {
```
Comment (dguy) [inline streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java]: This test doesn't seem to belong here. The test class is `InMemoryKeyValyLoggedStoreTest`, yet the test is `shouldCreatePersistentStore` If anything this should be moved to `StoresTest`, but maybe it is already covered

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5576/
Test FAILed (JDK 7 and Scala 2.11).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5562/
Test PASSed (JDK 8 and Scala 2.12).

File: streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java [hunk 3]
```diff
@@ -46,4 +52,14 @@
         store.init(context, store);
         return store;
     }
+
+    @Test
+    public void shouldPutAll() {
+        List<KeyValue<Integer, String>> entries = new ArrayList<>();
+        entries.add(new KeyValue<Integer, String>(1, "1"));
```
Comment (dguy) [inline streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java]: nit: can remove the type params from this line and the next

**Review by dguy — COMMENTED** [review]
Just one more minor comment.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5596/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5610/
Test PASSed (JDK 7 and Scala 2.11).

**Review by dguy — APPROVED** [review]
Thanks @jeyhunkarimov LGTM

**Pull request closed without merging**